### ISSUE TITLE
release version 0.9.7

### ISSUE THREAD

**Issue by GoogleCodeExporter** [NickstaDB/reflections#96]
```
There have recently been some changes in ZipDir to allow URLs with spaces in 
them (e.g. whose path contain %20).

I was planning to provide a patch myself, but looking at how the code looks 
now, I assume the problem is fixed.

It would be great if you could make a release as soon as possible, because this 
bug is killing me.

Thanks.
```

Original issue reported on code.google.com by `cvmoc...@gmail.com` on 16 Jan 2012 at 12:54

**Comment by GoogleCodeExporter** [comment]
```
Using revision 150 (0.9.7-SNAPSHOT, fix some issues):


I get the following exception when attempting to run org.reflections.VfsTest :


java.io.FileNotFoundException: 
C:\Program%20Files\Java\jdk1.6.0_27\jre\lib\ext\sunjce_provider.jar (The system 
cannot find the path specified)
    at java.io.FileInputStream.open(Native Method)
    at java.io.FileInputStream.<init>(FileInputStream.java:120)
    at com.google.common.io.Files$1.getInput(Files.java:100)
    at com.google.common.io.Files$1.getInput(Files.java:97)
    at com.google.common.io.ByteStreams.copy(ByteStreams.java:116)
    at com.google.common.io.Files.copy(Files.java:231)
    at com.google.common.io.Files.copy(Files.java:277)
    at org.reflections.VfsTest.test(VfsTest.java:43)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
    at java.lang.reflect.Method.invoke(Method.java:597)
    at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:44)
    at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
    at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:41)
    at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
    at org.junit.runners.BlockJUnit4ClassRunner.runNotIgnored(BlockJUnit4ClassRunner.java:79)
    at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:71)
    at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:49)
    at org.junit.runners.ParentRunner$3.run(ParentRunner.java:193)
    at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:52)
    at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:191)
    at org.junit.runners.ParentRunner.access$000(ParentRunner.java:42)
    at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:184)
    at org.junit.runners.ParentRunner.run(ParentRunner.java:236)
    at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:50)
    at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
    at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:467)
    at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:683)
    at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:390)
    at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:197)


```

Original comment by `mark.sor...@gmail.com` on 6 Mar 2012 at 9:16

**Comment by GoogleCodeExporter** [comment]
```
Forgot to add the following links that may help addressing this issue.

http://stackoverflow.com/questions/2166039/java-how-to-get-a-file-from-an-escape
d-url

URL url = getItSomehow();
File file = new File(url.toURI());

http://weblogs.java.net/blog/kohsuke/archive/2007/04/how_to_convert.html

```

Original comment by `mark.sor...@gmail.com` on 6 Mar 2012 at 9:36

**Comment by GoogleCodeExporter** [comment]
```
even an release candidate would be great (even if not on public maven repo).  
Right now, guava incompatibility is killing us :(
```

Original comment by `adrian.f...@gmail.com` on 25 Apr 2012 at 5:23

**Comment by GoogleCodeExporter** [comment]
```
released 0.9.7-RC1. comments?
```

Original comment by `ronm...@gmail.com` on 2 May 2012 at 3:51
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
fixed 0.9.7
```

Original comment by `ronm...@gmail.com` on 22 May 2012 at 7:47
- Changed state: **Fixed**
